data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/http/HttpConsumerVerticleFactory.java

```diff
@@ -199,7 +206,10 @@ private ConsumerRecordSender createConsumerRecordSender(
 
     final var circuitBreaker = CircuitBreaker
       .create(target, vertx, createCircuitBreakerOptions(egress))
-      .retryPolicy(computeRetryPolicy(egress));
+      .retryPolicy(computeRetryPolicy(egress))
+      .openHandler(r -> logger.info("Circuit breaker opened {}", keyValue("target", target)))
+      .halfOpenHandler(r -> logger.info("Circuit breaker half-opened {}", keyValue("target", target)))
+      .closeHandler(r -> logger.info("Circuit breaker closed {}", keyValue("target", target)));
```

Comment:
I guess these were added for debugging right? Don't forget to remove them before merging

Reply:
I'd like to keep them since it's a state change that is important to understand what's going on.

Reply:
Then enable them only in debug mode

Reply:
Why not keep them at info? It's quite useful to know when the circuit changes state.

Reply:
and it doesn't usually change too many times.